packages/neos-ui/src/Containers/Modals/DeleteNode/index.js

```diff
@@ -134,8 +137,12 @@ export default class DeleteNodeModal extends PureComponent {
                 id="neos-DeleteNodeDialog"
                 >
                 <div className={style.modalContents}>
-                    <I18n id="Neos.Neos:Main:content.navigate.deleteNodeDialog.header"/>
-                    &nbsp; {nodesToBeDeletedContextPaths.length > 1 ? `${nodesToBeDeletedContextPaths.length} ${i18nRegistry.translate('nodes', 'nodes', {}, 'Neos.Neos.Ui', 'Main')}` : `"${$get('label', node)}"`}?
+                    <p>
+                        <I18n id="Neos.Neos:Main:content.navigate.deleteNodeDialog.header"/>
+                        &nbsp; {nodesToBeDeletedContextPaths.length > 1 ? `${nodesToBeDeletedContextPaths.length} ${i18nRegistry.translate('nodes', 'nodes', {}, 'Neos.Neos.Ui', 'Main')}` : `"${$get('label', node)}"`}?
+                    </p>
+                    {warnings.length > 0 ? <hr /> : ''}
+                    {warnings.map((warning, index) => <p key={index}>{warning}</p>)}
```

Comment:
The warning should also be translated right?

Reply:
And how about showing the `node.label` so you know the context when there are multiple?

Reply:
node.label is very long sometimes, so it would look strange sometimes: 
![Bildschirmfoto 2022-12-13 um 14 13 15](https://user-images.githubusercontent.com/91674611/207328114-5388d5f0-920d-46f7-b2c8-c4598720b552.png)

I was thinking about grouping by nodeType and then using the nodeType label, what do you think? 

Translation is possible now, but using i18n in the yaml file currently does not work. I will check in the core for that

Reply:
How about a bullet list with items like this (untested Pseudocode):

`<NodeType.label>: <I>{String.cropAtWord(node.label, 100, '...'}</I>`

Reply:
I updated the label, it looks like this now: 

![Bildschirmfoto 2022-12-22 um 18 15 00](https://user-images.githubusercontent.com/91674611/209190083-b49f7fb5-6f12-49d0-9429-6bb435b82491.png)